src/plot.rs

```diff
@@ -166,7 +166,7 @@ impl Plot {
             y_tick_positions: [POS_NONE; NUMBER_OF_Y_AXES],
             y_tick_labels: [TICK_NONE; NUMBER_OF_Y_AXES],
             show_y_default_ticks: [false; NUMBER_OF_Y_AXES],
-            plot_flags: PlotFlags::NONE.bits() as sys::ImPlotFlags,
+            plot_flags: PlotFlags::ANTIALIASED.bits() as sys::ImPlotFlags,
```

Comment:
I agree, I always turn on antialiasing as well. However, this is a departure from the defaults that implot itself has - I believe there it was kept off by default because the author has seen it cause trouble in certain situations. I'm OK with making it the default here, but then we should add a note highlighting that. I'd add that to the help string for the method, something like `Note that this uses antialiasing by default, unlike the C++ API. If you are seeing artifacts or weird rendering, try disabling it.`

Reply:
agreed, should I just add that verbatim or would you like to do it yourself in some way? :)